python/graphscope/framework/graph.py

```diff
@@ -1068,49 +1056,102 @@ def _check_unmodified(self):
             self.signature == self._saved_signature, "Graph has been modified!"
         )
 
-    def save_to(self, path, **kwargs):
-        """Serialize graph to a location.
-        The meta and data of graph is dumped to specified location,
-        and can be restored by `Graph.load_from` in other sessions.
-
-        Each worker will write a `path_{worker_id}.meta` file and
-        a `path_{worker_id}` file to storage.
-        Args:
-            path (str): supported storages are local, hdfs, oss, s3
-        """
-
-        op = dag_utils.save_graph_to(self, path, self._vineyard_id, **kwargs)
-        self._session.dag.add_op(op)
-        return self._session._wrapper(op)
+    @staticmethod
+    def _load_from_graphar(path, sess, **kwargs):
+        # most configurations are set default for graphar.
+        vertex_map = utils.vertex_map_type_to_enum("global")
+        config = {
+            types_pb2.DIRECTED: utils.b_to_attr(True),
+            types_pb2.OID_TYPE: utils.s_to_attr("int64_t"),
+            types_pb2.GENERATE_EID: utils.b_to_attr(False),
+            types_pb2.RETAIN_OID: utils.b_to_attr(False),
```

Comment:
Why those configs are hardcoded. what if the actual graph config is different from these values? For example, the archived graph is a undirected one.

Reply:
Thanks for pointing out. Refined it. some parameter is not use in `graphar` loading and the oid_type and vid_type is fixed in `graphar`'s data.